RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1547,6 +1559,8 @@ class DeepTauId : public deep_tau::DeepTauBase {
                 << "invalid prediction = " << pred << " for tau_index = " << tau_index << ", pred_index = " << k;
           predictions.matrix<float>()(tau_index, k) = pred;
         }
+      } else {
+        continue;
```

Comment:
what is the purpose of this?

Reply:
I think that the intention was to ignore taus not passing discriminants, but for such a case anyway noting happens in the loop below, so I tend to agree that it is not needed. Is it correct?

Reply:
Indeed, it was. I copied these changes from a debugging repo and for these two lines I think I misread the use and need for them because when examining a bit closer now, it indeed seems like it is not needed. I will remove them.